packages/clerk-js/src/ui/signIn/utils.ts

```diff
@@ -101,3 +101,8 @@ export function determineSalutation(signIn: Partial<SignInResource>): string {
 
   return titleize(signIn.userData?.firstName) || titleize(signIn.userData?.lastName) || signIn?.identifier || '';
 }
+
+const supportedStrategies: SignInStrategy[] = ['email_code', 'password', 'phone_code', 'email_link'];
+export function isContactInfoRelatedStrategy(strategy: SignInStrategy): boolean {
```

Comment:
🙃 
```suggestion
export function hasLocalAuthenticationfactors(strategy: SignInStrategy): boolean {
```

Reply:
🙃  I'd make it a `factorHasLocalStrategy` and make it accept a `SignInFactor`

Reply:
I like this one, SignInFactor doesn't match the string that I want to pass

Reply:
Not sure I quite understand, sorry!

Is this type: https://github.com/clerkinc/javascript/blob/276ef8e7a332f8604ef9956af6386db14187a68a/packages/types/src/signIn.ts#L95-L94 not working for you?

What I had in mind was something closer to: 
```ts
export function factorHasLocalStrategy(factor: SignInFactor): boolean 
```